doc/dev/mgmt/tests.md

```diff
@@ -15,58 +15,43 @@ IMPORTANT NOTE: All the commands in this page assumes you have loaded the [dev_s
 
 # Overview
 
-This page is to help you write tests for Azure Python SDK when these tests require Azure HTTP requests.
-The Azure SDK test framework uses the [`azure-devtools`](https://pypi.python.org/pypi/azure-devtools) package,
-which in turn rests upon on a HTTP recording system ([vcrpy](https://pypi.python.org/pypi/vcrpy))
-that enables tests depending on network interaction
-to be run offline.
+This page is to help you write tests for Azure Python SDK when these tests require Azure HTTP requests. The Azure SDK test framework uses the [`azure-devtools`](https://pypi.python.org/pypi/azure-devtools) package, which in turn rests upon on a HTTP recording system ([vcrpy](https://pypi.python.org/pypi/vcrpy)) that enables tests dependent on network interaction to be run offline.
 
 In this document, we will describe:
--   How to run the tests offline, using previously recorded HTTP interactions,
-    or online, by authenticating with Azure to record new HTTP interactions
+-   How to run the tests offline (using previously recorded HTTP interactions)
+-   How to run the tests online (by authenticating with Azure to record new HTTP interactions)
 -   How to write new tests using our utility classes
 
 # Getting the tests to run
 
-This section describes how to run the SDK tests,
-by installing dependencies into a virtual environment
-and using a helper script and the [pytest](https://docs.pytest.org/en/latest/) test runner
-to choose which tests to run.
+This section describes how to run the SDK tests, by installing dependencies into a virtual environment and using a helper script and the [pytest](https://docs.pytest.org/en/latest/) test runner to choose which tests to run.
 
 ## Running the tests
 
-Azure SDK tests use [pytest](https://docs.pytest.org/en/latest/) test runner.
-To run all the tests, you can use the following command:
+Azure SDK tests use [pytest](https://docs.pytest.org/en/latest/) test runner. To run all the tests, you can use the following command:
 
 ```Shell
 pytest
 ```
 
-You can provide directories or individual files as positional arguments
-to specify particular tests to run
-rather than running the entire test suite. For example:
+You can provide directories or individual files as positional arguments to specify particular tests to run rather than running the entire test suite. For example:
 ```Shell
-pytest -s sdk/storage/azure-mgmt-storage/
+pytest sdk/storage/azure-mgmt-storage/
```

Comment:
I kinda like -s, since otherwise new devs get confused with "why aren't my debug prints showing up" (literally had this exact conversation the other day).  Will default to you/laurent, of course, but that's my thought.

Reply:
I removed it here because it's implied that -s is how you pass in specific directories. I will add a section that -s will pass print statements to standard output.